woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2044,7 +2050,8 @@ def test_setitem_new_column(sample_df):
     assert 'test_col2' in df.columns
     assert 'test_col2' in df.ww._schema.columns.keys()
     assert df.ww['test_col2'].ww.logical_type == Integer
-    assert df.ww['test_col2'].ww.semantic_tags == set()
+    assert df.ww['test_col2'].ww.use_standard_tags is True
+    assert df.ww['test_col2'].ww.semantic_tags == {'numeric'}
```

Comment:
Note the change in behavior here:
The dataframe was initialized with `use_standard_tags=Falst`, but when we use `setitem` with a pandas series, we have to init Woodwork, so we default to `use_standard_tags=True` since we have no overall table value.

Users will have to init woodwork on the series to get `use_standard_tags` to be something other than the default.

Reply:
I could potentially see this as a point of confusion, but I think this behavior makes sense. If users want to control what Woodwork does, they should initialize WW first and specify what they want, otherwise they get the default WW behavior.